ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -581,6 +585,44 @@ def test_ipa_healthcheck_no_errors(self):
         )
         assert returncode == 0
 
+    def test_ipa_healthcheck_no_errors_with_overrides(self):
+        """
+        Test overriding command-line options in a configuration file.
+        """
+        tmpcmd = self.master.run_command(['mktemp'])
+        config_file = tmpcmd.stdout_text.strip()
+
+        self.master.put_file_contents(
+            config_file,
+            '\n'.join([
+                '[default]',
+                'output_type=human'
+            ])
+        )
+        returncode, output = run_healthcheck(
+            self.master, failures_only=True, config=config_file
+        )
+        assert returncode == 0
+        assert output == "No issues found."
+
+        # Setting an output file automatically enables all=True
+        self.master.put_file_contents(
+            config_file,
+            '\n'.join([
+                '[default]',
+                'output_type=human',
+                'output_file=/tmp/hc.log',
+            ])
+        )
+        returncode, _unused = run_healthcheck(
+            self.master, config=config_file
+        )
+        logsize = len(self.master.get_file_contents('/tmp/hc.log'))
+        self.master.run_command(['rm', '-f', '/tmp/c.log'])
```

Comment:
The file name has a typo (`c.log` should be `hc.log`). Probably safer to define a constant for the file name.

Reply:
ok

Reply:
Fixed.